pkg/eventsapi/eventsapi.go

```diff
@@ -125,6 +136,9 @@ func enqueueEvent(context context.Context, client *http.Client, url string, even
 	}
 
 	_ = json.Unmarshal(respBody, &response)
+	if isSuccess(httpResp, err) {
```

Comment:
@dobs: Instead of explicitly setting a "success" / "fail" here, we could instead return an error (which could get wrapped up in `eventqueue.Response`)

Reply:
Alternate solution here that may end up being a bit cleaner here: https://github.com/PagerDuty/go-pdagent/pull/9